common/achievement/bucket.spec.ts

```diff
@@ -0,0 +1,280 @@
+import moment from 'moment';
+import { bucketCreatorDefs } from './bucket';
+import { BucketCreatorContext, ContextMatch, ContextSubcourse, TimeBucket } from './types';
+
+describe('test create buckets by_lecture_start', () => {
+    // TODO: think about setting dow globally
+    moment.updateLocale('de', { week: { dow: 1 } });
```

Comment:
I think we should set `dow: 1` globally. Otherwise, the week is starting on sundays.